.github/workflows/start-ci.yml

```diff
@@ -0,0 +1,78 @@
+name: Label to staging
+
+on:
+  pull_request_target:
+    types: [labeled]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+
+      - uses: actions/checkout@v2
+        if: github.event.label.name == 'start-ci'
+        with:
+          ref: ${{ github.event.pull_request.head.sha }}
+
+      - name: Setup GCloud
+        if: github.event.label.name == 'start-ci'
+        uses: GoogleCloudPlatform/github-actions/setup-gcloud@master
+        with:
+          project_id: ${{ secrets.GCP_PROJECT_ID }}
+          service_account_email: ${{ secrets.GCP_SA_EMAIL }}
+          service_account_key: ${{ secrets.GCP_SA_KEY }}
+          export_default_credentials: true
+
+      - uses: actions/cache@v1
+        if: github.event.label.name == 'start-ci'
+        with:
+          path: ~/.npm
+          key: ${{ runner.os }}-node-${{ hashFiles('**/package-lock.json') }}
+          restore-keys: |
+            ${{ runner.os }}-node-
+
+      - name: Install Dependencies
+        if: steps.cache.outputs.cache-hit != 'true' && steps.comment.outputs.result == 'true'
```

Comment:
shouldn't `npm ci` always run, regardless of cache existing or not?

Reply:
The cache key is based on the lock file, so if there are no changes to that, the ci call isn't needed. Although that's not the case for any repos that have install hooks for building.
I know this is broken for the issue version of this https://github.com/nodejs/nodejs.dev/runs/981741342?check_suite_focus=true#step:21:2 and could probably be removed

Reply:
Decided just to drop both conditions, since I don't think it's going to affect speed much. I'll see if this gives a similar error after landing